src/Uno.UWP/Media/Playback/MediaPlayer.wasm.cs

```diff
@@ -0,0 +1,297 @@
+﻿using System;
+using Uno.Media.Playback;
+using Windows.Media.Core;
+using Uno.Extensions;
+using System.IO;
+using Uno.Foundation.Logging;
+using System.Collections.Generic;
+using Uno;
+using Uno.Helpers;
+using System.Linq;
+
+namespace Windows.Media.Playback
+{
+	public partial class MediaPlayer
+	{
+		public IHtmlMediaPlayer Player;
```

Comment:
If intended to be public, should be a property. Otherwise, make private and rename to `_player`.